packages/stores/types.ts

```diff
@@ -219,6 +261,18 @@ type TNotification =
     | ((withdrawal_locked: boolean, deposit_locked: boolean) => TNotificationMessage)
     | ((excluded_until: number) => TNotificationMessage);
 
+type LoginParams = {
+    acct: string;
+    token: string;
+    curr: string;
+    lang: string;
+};
+
+type IncrementedProperties<N extends number> = {
+    [K in keyof LoginParams as `${string & K}${N}`]: string;
+};
```

Comment:
@farzin-deriv if we want to change IncrementProperties in this case we have to change LoginURLParams too, because we have to pass 2 parameters to generic, is it ok:

> type IncrementedProperties<O extends object, N extends number> = {
>     [K in keyof O as `${string & K}${N}`]: string;
> };
> 
> type LoginURLParams<N extends number> = LoginParams & IncrementedProperties<LoginParams, N>;